src/ui/sign_tx_nbgl.c

```diff
@@ -0,0 +1,284 @@
+#include <stdbool.h>  // bool
+#include <string.h>   // memset
+
+#include "os.h"
+#include "ux.h"
+#include "glyphs.h"
+
+#include "constants.h"
+#include "../globals.h"
+#include "../io.h"
+#include "../sw.h"
+#include "action/validate.h"
+#include "../transaction/types.h"
+#include "../common/format.h"
+#include "utils.h"
+#include "menu.h"
+#include "../shared_context.h"
+#include "sign_tx_common.h"
+
+#ifdef HAVE_NBGL
```

Comment:
Why not put the guard at the top of the file ?

Reply:
Why not